keps/sig-node/2133-kubelet-credential-providers/kep.yaml

```diff
@@ -27,13 +27,12 @@ stage: beta
 # The most recent milestone for which work toward delivery of this KEP has been
 # done. This can be the current (upcoming) milestone, if it is being actively
 # worked on.
-latest-milestone: "v1.23"
+latest-milestone: "v1.24"
 
 # The milestone at which this feature was, or is targeted to be, at each stage.
 milestone:
   alpha: "v1.20"
-  beta: "v1.23"
-  stable: "v1.25"
+  beta: "v1.24"
```

Comment:
I don't think much has changed since adding the Beta graduation criteria in v1.23, so this should just be a matter of updating the milestones. 

@DangerOnTheRanger @endocrimes @adisky were there any recent updates on this effort worth noting in the KEP?

Reply:
None come to mind to me!

Reply:
@andrewsykim yes nothing has changed, Thanks for the update!!